Automattic/wp-calypso
# Preload setup page on the authorize page

Related to #96420

## Proposed Changes

* This PR preloads SetupImage, which is the first image used on the Woo Loader, to prevent text from shifting when the image is loaded.


## Testing Instructions

1. Checkout this branch locally and run `yarn start`
2. Make sure you're logged-in on WPCOM.
3. Visit this [link](http://calypso.localhost:3000/jetpack/connect/authorize?client_id=239172539&redirect_uri=https%3A%2F%2Fpink-piccolo.jurassic.ninja%2Fwp-admin%2Fadmin.php%3Fhandler%3Djetpack-connection-webhooks%26action%3Dauthorize%26_wpnonce%3Df3099546f0%26redirect%3Dhttps%253A%252F%252Fpink-piccolo.jurassic.ninja%252Fwp-admin%252Fadmin.php%253Fpage%253Dwc-admin&state=1&scope=administrator%3A1a2e2b51282f8ac685ca15e67964f8f4&user_email=moon.kyong%40automattic.com&user_login=moon0326&jp_version=14.1-a.3&secret=QYBTIYTgqLIIMcpg68zLcFnH8pyT9Jmf&blogname=Pink%20Piccolo&site_url=https%3A%2F%2Fpink-piccolo.jurassic.ninja&home_url=https%3A%2F%2Fpink-piccolo.jurassic.ninja&site_icon=&site_lang=en_US&site_created=2024-11-19%2018%3A10%3A55&allow_site_connection=1&calypso_env=production&source=&_as=wp-cli&locale=en&from=woocommerce-core-profiler&plugin_name=woocommerce-payments&color_scheme=sunrise&_ui=189042142&_ut=wpcom%3Auser_id&purchase_nonce=3UiGhvFH&_wp_nonce=d1d5864992&redirect_after_auth=https%3A%2F%2Fpink-piccolo.jurassic.ninja%2Fwp-admin%2Fadmin.php%3Fpage%3Dwc-admin&site=https%3A%2F%2Fpink-piccolo.jurassic.ninja)
4. Click on `Connect to WordPress.com` and pay attention to the text.
5. Make sure text and the image load at the same time.

## Pre-merge Checklist
